ui-backend/catalog-ui-search/src/main/resources/OSGI-INF/metatype/org.codice.ddf.catalog.ui.attribute.xml

```diff
@@ -44,10 +44,23 @@ Note: Attributes must also be listed under the Facet Attribute Whitelist for thi
             />
     </OCD>
 
+    <OCD name="Catalog UI Search Featured Attributes" id="org.codice.ddf.catalog.ui.attribute.featuredAttributes">
```

Comment:
Will there be any issues with calling these 'featured attributes' everywhere but on the group heading? IMO, 'featured' describes this concept better than 'common'.

Reply:
I personally would keep it all consistent, although I agree "featured" is a better descriptor.

Reply:
Changed everything to 'common'.